packages/fuels-test-helpers/src/script.rs

```diff
@@ -0,0 +1,31 @@
+use fuel_core::service::{Config, FuelService};
+use fuel_gql_client::client::FuelClient;
+use fuel_tx::{consts::MAX_GAS_PER_TX, Transaction};
+use fuels_contract::script::Script;
+use std::fs::read;
+
+#[cfg(feature = "fuels-signers")]
+pub async fn script_runner(bin_path: &str) -> u64 {
```

Comment:
This should be put in `lib.rs` in the same directory IMO

Reply:
In lib.rs we have setups, in signers.rs we have launchers, so I thought it would be nicer to make script.rs in which we will have everything related to scripts.

Reply:
Ok I'm fine with it.